index.html

```diff
@@ -46,717 +46,789 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
+                                        Language in Robotic Affordances. </a> Ahn et al. (2022)</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><b>Week 8</b><br />
+                            Fri, May 23
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a" href="slides/cs224r_rl_robots_reset_free.pdf">RL for Robots: Reset-Free RL</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i> <b>Milestone Due</b><br>
+                        </td>
+                        <td>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 9</b><br />
+                            Wed, May 28
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a" href="slides/cs224r_rl_robots_sim2real.pdf">RL for Robots: Sim-to-Real Transfer</a>
+                        </td>
+                        <td>
+
+                        </td>
+                        <td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 9</b><br />
+                            Fri, May 30
+                        </td>
+                        <td><i><span class="label label-guest text-base">Guest Lecture</span></i>
+                            <a" href="slides/cs224r_fronteirs.pdf">Frontiers</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i><b> HW4</b><br>
+                        </td>
+                        <td>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 10</b><br />
+                            Wed, June 4
+                        </td>
+                        <td><i><span class="label label-reading text-base">Presentations</span></i>
+                            No Lecture (Poster Session)</td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i> <b>Poster Due</b><br>
+                        </td>
+                        <td>
+                            <!-- <ul>
+                                <li>The poster session will be held at the Gates AT&T Lawn from 4-7pm.</li>
+                            </ul> -->
+                        </td>
+                    </tr>
+
+                    <tr class="active">
+                        <td><b>Week 11</b><br />
+                            Fri, June 6
+                        </td>
+                        <td>
+                            No Lecture
```

Comment:
Maybe delete this row? It's the start of finals, so it should be clear that there isn't a lecture

Reply:
(also this is technically still week 10, not week 11, but that's not relevant if this is deleted)